app/Models/User.php

```diff
@@ -2,46 +2,70 @@
 
 namespace App\Models;
 
-// use Illuminate\Contracts\Auth\MustVerifyEmail;
```

Comment:
Why would remove all these user traits?

Reply:
The `Orchid` `Orchid\Platform\Models\User` model extends `Authenticatable`  which still has those traits